src/plots/piePlot.ts

```diff
@@ -213,7 +213,7 @@ export module Plots {
       var center = { x: this.width() / 2, y: this.height() / 2 };
       var adjustedQueryPoint = { x: queryPoint.x - center.x, y: queryPoint.y - center.y };
       var radius = Math.sqrt(Math.pow(adjustedQueryPoint.x, 2) + Math.pow(adjustedQueryPoint.y, 2));
-      var angle = Math.acos(-adjustedQueryPoint.y / (1 + radius));
```

Comment:
https://github.com/palantir/plottable/pull/2340#discussion-diff-32654964

...I added instead of multiplied the two magnitudes...